src/tests/ftest/util/command_utils.py

```diff
@@ -1,5 +1,5 @@
 """
-  (C) Copyright 2018-2024 Intel Corporation.
+  (C) Copyright 2018-2025 Intel Corporation.
```

Comment:
Intel copyright should not be updated anymore. You'll need to merge latest master, manually revert this to 2024, make sure githooks are installed, and let the hook update for you. Ultimately it should look like
```suggestion
  (C) Copyright 2018-2024 Intel Corporation.
  (C) Copyright 2025 Hewlett Packard Enterprise Development LP
```

For files not modified in 2025, the GitHub Action check will still complain because it is not smart enough to distinguish between 2024 commits and 2025 commits, so I think we can just ignore those for files only modified in 2024

To setup the githooks: `git config core.hookspath utils/githooks`

Reply:
I did merge the latest master. Not sure if it took all proper changes or not.